bundles/org.openhab.binding.modbus.sunspec/src/main/java/org/openhab/binding/modbus/sunspec/internal/SunSpecConstants.java

```diff
@@ -33,17 +33,27 @@
     // List of all Thing Type UIDs
     public static final ThingTypeUID THING_TYPE_INVERTER_SINGLE_PHASE = new ThingTypeUID(BINDING_ID,
             "inverter-single-phase");
+    public static final ThingTypeUID THING_TYPE_INVERTER_SPLIT_PHASE = new ThingTypeUID(BINDING_ID,
+            "inverter-split-phase");
+    public static final ThingTypeUID THING_TYPE_INVERTER_THREE_PHASE = new ThingTypeUID(BINDING_ID,
+            "inverter-three-phase");
 
     // Block types
     public static final int COMMON_BLOCK = 1;
     public static final int INVERTER_SINGLE_PHASE = 101;
+    public static final int INVERTER_SPLIT_PHASE = 102;
+    public static final int INVERTER_THREE_PHASE = 103;
     public static final int FINAL_BLOCK = 0xffff;
 
     /**
      * Map of the supported thing type uids, with their block type id
      */
-    public static final Map<Integer, ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Collections
-            .singletonMap(INVERTER_SINGLE_PHASE, THING_TYPE_INVERTER_SINGLE_PHASE);
+    public static final Map<Integer, ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = new HashMap<>();
```

Comment:
Does this need to be a map? I don't see you using the map keys, is that for a future PR?

Reply:
This is already in use, the keys are checked by the SunspecDiscoveryProcess to recognize the block level id. See:
https://github.com/openhab/openhab-addons/blob/47a320c1d745eb65c3354b28c4f1a278425eadb7/bundles/org.openhab.binding.modbus.sunspec/src/main/java/org/openhab/binding/modbus/sunspec/internal/discovery/SunspecDiscoveryProcess.java#L327-L330

Background: in the specification device types are identified by a unique identifer for example: 101 - 1 phase inverter, 102 - 2 phase inverter, 103 - 3 phase inverter, 201 - 1 phase meter, etc.
SUPPORTED_THING_TYPES_UIDS is a map between those identifiers and the OH thing type uids.